src/asmcnc/production/lowerbeam_calibration_jig/lowerbeam_calibration_success.py

```diff
@@ -62,8 +63,7 @@ def enter_prev_screen(self):
         self.sm.current = 'lbc4'
 
     def shutdown_console(self):
```

Comment:
Why have this function in the first place? just call console_utils.shutdown() directly